data/korath/korath missions.txt

```diff
@@ -257,3 +257,274 @@ mission "Small Scale Delivery"
+		conversation
+			`It takes you some time to find the <EDC-ship>. You had expected a beacon of some sort but they are either keeping radio silence to avoid drawing the Mereti or there were no survivors.`
+			`	When you manage to open the hatch, you find yourself staring down a thermal rifle. The Korath shouts something at you and you hold your hands out palms forward. He limps back a few steps but keeps the weapon trained on you. You slowly pull your communicator out and play your ship's recording of the Quarg transmission. When the Korath hears the Quarg's voice they become excited. "<planet>?" he asks repeatedly. You nod your head and show your palms again.`
+			`	He leads you into the ship, shouting into a communication device that is then picked up on speakers throughout the ship. You can only glance around as you move through the engineering section but it's clear they are running on emergency power. You notice that their Systems Core was destroyed, leaving them stranded.`
+			`	The living area is a makeshift hospital. You don't see any bodies, but there are far fewer Korath here than you would expect to crew a ship of this size. It's clear that all the surviving crew are injured to some degree with more than half incapacitated or unconscious. Even if the ship is spaceworthy, there aren't enough crew to fly it.`
+			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. You call your crew over to help move the survivors to the <ship>. Within a couple of hours you are ready to go.`
+				to display
+					"flagship crew" > 1
+			`	You quickly count the number of survivors and communicate to your guide that you can take them all. He mimics your nodding head gesture, shows you his palms and makes quick circles. You think he is agreeing and wants to hurry. With no crew helping you, it takes several hours to transfer them to the <ship>.`
+				to display
+					"flagship crew" <= 1
+	
+	on complete
+		payment 300000
+		conversation
+			`The Efreti commander hails the authorities once you enter the system and they have medical crews waiting when you land. It takes several hours to evacuate the injured as doctors or maybe paramedics swarm over them to triage, treat, and prepare for transport. The three who died during the flight are treated with respect and carried off the <ship>.`
+			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to seek out those where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

Comment:
```suggestion
			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to find those where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

Reply:
No, this changes the meaning. "Seek out" just means you're trying to find it. "Find" means you found it. The Quarg are expressing the hopelessness of the search. Those Efret are looking for things they know they won't find, because they already know their worlds are dead.

Reply:
How about this?
```suggestion
			`	When you make it to the spaceport, you find a message from the Quarg. "Bold human! With profuse thanks we offer you for helping the stranded Korath. The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers to seek what they can salvage among the ruins, and to search for those ruins where they may one day live again. For the young, this gives them hope, but sometimes they are caught and the Efreti are reminded why they live crushed together around these few stars."`
```

Reply:
They're searching for worlds to live on, and ruins in which to salvage. That run-on sentence is confusing.

Reply:
How about this?

> The Korath ever mourn the loss of their worlds, and, from time to time, brave the dangers. They salvage what they can among the ruins, and seek out worlds where they may one day live again.